_dashboards/reporting-cli.md

```diff
@@ -0,0 +1,248 @@
+### Install the Reporting CLI from NPM
+
+To download the .tar archive and install the reporting tool with NPM, run the following command.
+
+To download the Reporting CLI .tar archive and install it, run the following command.
+
+```
+npm install opensearch-reporting-cli
+```
+
+### Install the Reporting CLI from OpenSearch.org
+
+You can download the `opensearch-reporting-cli` tool from the OpenSearch.org Downloads page, see [Command Line Tools](https://opensearch.org/downloads.html#command-line-tools).
+
+Next, run the following command to install the .tar archive.
+
+```
+npm install -g opensearch-reporting-cli.tar.gz
+```
+
+## Create and request a visualization report
+
+First, you need to get the URL for the Dashboards visualization that you want to download as an image file or PDF.
+
+To generate a visualization report, you need to specify the Dashboards URL.
+
+Open the Dashboards visualization of which you want to generate a report, and select **Share >  Permalinks > Generate link as Shapshot > Short URL > Copy link**.
+
+![Copy link]({{site.url}}{{site.baseurl}}/images/dashboards/dash-url.png)
+
+You will need to add the URL with the `-u` argument when you request the report in the CLI.
+
+### Example: Request a PNG file
+
+The following command shows how to request an OpenSearch Dashboards report in PNG format with basic authentication and send the report via SES to an email address:
+
+```
+opensearch-reporting-cli -u https://search-basic-auth2-3-b477rrluhlckrmwmtvp6k3xxsi.us-west-2.es.amazonaws.com/_dashboards/goto/069af6d6f3294421ec163b07fef91e5d?security_tenant=global -a basic -c admin:1234 -e ses -s user@amazon.com -r user@amazon.com -f png
+```
+
+### Example: Request a PDF file
+
+The following command requests a PDF file and specifies the email address for the recipient.
+
+```
+opensearch-reporting-cli -u https://localhost:5601/app/dashboards#/view/7adfa750-4c81-11e8-b3d7-01146121b73d -a basic -c admin:Test@1234 -e ses -s user@domain.com -r user@domain.com -f pdf
+```
+
+Upon success, the file will be sent to the specified email address.
+
+![PDF example]({{site.url}}{{site.baseurl}}/images/dashboards/cli-pdf-report.png)
+
+
+### Example: Request a CSV file
+
+The following command generates a report that contains all table content in CSV format, and sends the report using SES transport to an email address.
+
+```
+opensearch-reporting-cli -u https://search-basic-auth2-3-b477rrluhlckrmwmtvp6k3xxsi.us-west-2.es.amazonaws.com/_dashboards/goto/f166434919d2d8d3708a4705b6f2ed2d?security_tenant=global -f csv -a basic -c admin:Test1234 -e ses -s user@amazon.com -r user@amazon.com
+```
+
+Upon success, you will receive the email with the CSV file attached.
+
+<!--Question for eng: Is the rest of the command similar to the other examples with exception of "-a cognito" specified for authentication? If so, perhaps we don't need this example.
```

Comment:
sure, we can skip this example. Supported auth options are already mentioned above.